test/server/cards/06-CotE/KakitaToshimoko.spec.js

```diff
@@ -0,0 +1,231 @@
+                this.initiateConflict({
+                    attackers: [this.kakitaToshimoko],
+                    defenders: [this.akodoToturi2],
+                    type: 'military'
+                });
+                this.noMoreActions();
+                expect(this.player1).toHavePrompt('Triggered Abilities');
+                expect(this.player1).toBeAbleToSelect(this.kakitaToshimoko);
+                this.player1.clickCard(this.kakitaToshimoko);
+                this.player2.clickCard(this.akodoToturi2);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.kireiKo);
+                this.player2.clickCard(this.kireiKo);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.stayYourHand);
+                this.player2.clickCard(this.stayYourHand);
+                expect(this.getChatLogs(1)).toContain('player2 won a military conflict 6 vs 0');
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.fallenInBattle);
+                this.player2.clickCard(this.fallenInBattle);
+                this.player2.clickCard(this.kakitaToshimoko);
+                expect(this.kakitaToshimoko.location).toBe('dynasty discard pile');
+                expect(this.player1).toHavePrompt('Action Window');
+            });
+
+            it('should still recalculate conflict totals even if the duel is canceled (Shiba Yojimbo)', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.kakitaToshimoko],
+                    defenders: [this.akodoToturi2, this.adeptOfTheWaves],
+                    type: 'military'
+                });
+                this.noMoreActions();
+                expect(this.player1).toHavePrompt('Triggered Abilities');
+                expect(this.player1).toBeAbleToSelect(this.kakitaToshimoko);
+                this.player1.clickCard(this.kakitaToshimoko);
+                this.player2.clickCard(this.adeptOfTheWaves);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.shibaYojimbo);
+                this.player2.clickCard(this.shibaYojimbo);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.kireiKo);
+                this.player2.clickCard(this.kireiKo);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                this.player2.clickPrompt('Pass'); //Stay Your Hand
+                expect(this.getChatLogs(1)).toContain('player2 won a military conflict 8 vs 0');
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.fallenInBattle);
```

Comment:
This, for some reason, would not work even with Stay Your Hand merged.  Fallen in Battle would not trigger. ??